pytorch_lightning/accelerators/cpu.py

```diff
@@ -19,8 +19,17 @@
 
 from pytorch_lightning.accelerators.accelerator import Accelerator
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _PSUTIL_AVAILABLE
 from pytorch_lightning.utilities.types import _DEVICE
 
+if _PSUTIL_AVAILABLE:
+    import psutil
+
+# CPU device metrics
+_CPU_VM_PERCENT = "cpu_vm_percent"
```

Comment:
why define them as global variables? I don't see them used anywhere else.

Reply:
They are used by the tests